internal/utils/main.go

```diff
@@ -593,6 +593,13 @@ func (e EnvironmentDetails) GetLedgerCloseMeta(end uint32) (xdr.LedgerCloseMeta,
 
 	backend, err := e.CreateCaptiveCoreBackend()
 
+	ledgerRange := ledgerbackend.UnboundedRange(end)
```

Comment:
We should be using a `BoundedRange` with the `export_ledger_entry_changes` command utility since that utility passes both a _start_ and _end_ bound. This would pull all of history from genesis onward, even though we would discard the majority of it.